### DISCUSSION TITLE
「绕过中国大陆IPv4黑名单」到底是指什么？

### DISCUSSION THREAD

**Discussion by Rangtian** [vernesong/OpenClash#3605] [Q&A] [answered]
「列表中的域名或 IP 将不会受到绕过中国大陆 IP 选项的影响，此功能依赖于 Dnsmasq 」没解释清楚啊！
这是指国内域名、IP强制走代理？
还是指国外域名、IP强制不走代理？

**Comment by vernesong** [comment, marked as answer]
正常进clash按照规则分流

**Reply by Rangtian** [reply]
感谢大佬耐心回复！

**Comment by filwu8** [comment]
请教，不知道这么理解对不：填在这里的ip地址，在进入clash核心之前，就会被判定是国内ip，因而不会走代理？
![1729675908418](https://github.com/user-attachments/assets/eba2ab3c-94ef-493d-a0a9-19b3d362f7e1)

**Reply by filwu8** [reply]
比如说之前，steam下载游戏，好像强制走代理了，只有2m每秒，自从 steamcontent.com 这个地址填到这个位置以后，下载好像走了国内CDN，飙升到300M每秒

**Reply by refined-fish** [reply]
你理解反了吧，这个是绕过大陆功能的“黑名单”。意思是：你开启绕过功能后，本来匹配的会被饶过功能给绕过Clash核心，然后直连；但是加入黑名单的会被绕过功能排除，也就是不绕过直接进入Clash核心。

你如果启用的是海外绕过模式，那么就效果相反
